plugins/aws/iam/iamRolePolicies.js

```diff
@@ -3,34 +3,61 @@ var helpers = require('../../../helpers/aws');
 
 var managedAdminPolicy = 'arn:aws:iam::aws:policy/AdministratorAccess';
 
+function hasFederatedUserRole(policyDocument) {
+    // true iff every statement refers to federated user access
+    let statement;
+    for (statement of policyDocument.Statement) {
+        if (statement.Action !== 'sts:AssumeRoleWithSAML' && statement.Action !== 'sts:AssumeRoleWithWebIdentity'){
+            return false;
+        }
+    }
+    return true;
+}
+
```

Comment:
can you move this function to our helpers file: https://github.com/cloudsploit/scans/blob/master/helpers/aws/functions.js
and just import the file like this: `var helpers = require('../../../helpers/aws');`